authorization.go

```diff
@@ -84,6 +89,27 @@ func (c *AuthorizationClient) GetList(ctx context.Context, paymentID string) ([]
 	return authorizations, nil
 }
 
+// ContinueAuthentication continues the authentication flow for the specified payment ID and authorization ID.
+// Returns nil if everything is ok, and error when continue flow failed
+func (c *AuthorizationClient) ContinueAuthentication(ctx context.Context, idempotencyKey, paymentID, authorizationID string, params *ContinueAuthenticationParams, clientInfo *ClientInfo) error {
+	headers := map[string]string{headerIdempotencyKey: idempotencyKey}
+
+	if clientInfo != nil {
+		headers[headerClientIPAddress] = clientInfo.IPAddress
+		headers[headerClientUserAgent] = clientInfo.UserAgent
+	}
+
+	if err := c.Caller.Call(ctx, "POST", c.authenticationPath(paymentID, authorizationID), headers, params, nil); err != nil {
```

Comment:
I suggest that we return the updated authorization object from this method as well. 
We will need it in the code anyway and this allows not to do an extra API call. .
It can be extracted from the related_resources response field.

Reply:
I didn't get the response authorization object, because we have array of objects in response. Of course we can take the first element, but its not good idea from my point of view. What do you think?

Reply:
We know the authorization id, so we can intelligently pick the right object with the same id from the list

Reply:
added